sdk/communication/azure-communication-chat/README.md

```diff
@@ -80,7 +80,9 @@ The id for the user created above will be necessary later to add said user as a
 
 ## Chat Thread
 
-A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat. Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+A chat conversation is represented by a chat thread. Each user in the thread is called a thread member. Thread members can chat with one another privately in a 1:1 chat or huddle up in a 1:N group chat.
+
+Users also get near real-time updates for when others are typing and when they have read the messages. To learn more, read about chat concepts [here](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
```

Comment:
We do not have real-time update yet. Instead let's say "Using the APIs, users can also send typing indicators when typing a message and read receipts for the messages they have read in a chat thread. To learn more, read about chat service here." and then link here to the chat concept doc

Reply:
"and read receipts for the messages they have read". Do you mean "and **send** receipts for the messages they have read"?